scripts/west_commands/run_common.py

```diff
@@ -127,6 +127,8 @@ def add_parser_common(command, parser_adder=None, parser=None):
         etc.'''))
     group.add_argument('-H', '--context', action='store_true',
                        help='print runner- and build-specific help')
+    group.add_argument('-f', '--force', action='store_true',
+                       help='force download with command line ignore load error')
```

Comment:
Can you please refactor this routine so that only `flash` has this option? It looks like `debug` and `debugserver` do not support this, so I don't think they should have the option.

Reply:
good point, thanks, refactor.